<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Socialgram</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="sign_up">
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>381</width>
        <height>301</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="SPsignupButton">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>220</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Sign Up</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Sign Up</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>191</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextEdit" name="SPusername">
          <property name="placeholderText">
           <string>Enter a username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="SPpassword">
          <property name="placeholderText">
           <string>Create a password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="SPpasswordConfirm">
          <property name="placeholderText">
           <string>Confirm password</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>250</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Already have an account ?</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SPloginButton">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>250</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="login">
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>210</y>
        <width>761</width>
        <height>481</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="sign_up_2"/>
      <widget class="QWidget" name="login_2"/>
      <widget class="QWidget" name="home_2">
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>30</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Home Page</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>381</width>
        <height>301</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="LloginButton">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>220</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Log In</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>191</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="Lusername">
          <property name="placeholderText">
           <string>Enter your email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="Lpassword">
          <property name="placeholderText">
           <string>Enter you password</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>56</x>
         <y>250</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Don't have an account ? </string>
       </property>
      </widget>
      <widget class="QPushButton" name="LsignupButton">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>250</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Sign Up</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="home">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>0</y>
        <width>377</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="settingsButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="app-images.qrc">
        <normaloff>:/resources/settings-icon.jpg</normaloff>:/resources/settings-icon.jpg</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="messageButton">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>0</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="app-images.qrc">
        <normaloff>:/resources/messageIcon.jpg</normaloff>:/resources/messageIcon.jpg</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="profileButton">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>0</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="app-images.qrc">
        <normaloff>:/resources/profile-icon.webp</normaloff>:/resources/profile-icon.webp</iconset>
      </property>
     </widget>
     <widget class="QScrollArea" name="newsFeed">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>761</width>
        <height>421</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>745</width>
         <height>419</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="app-images.qrc">
        <normaloff>:/resources/search-icon.png</normaloff>:/resources/search-icon.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="addPostButton">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>0</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="app-images.qrc">
        <normaloff>:/resources/addPost.png</normaloff>:/resources/addPost.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="messages">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>0</y>
        <width>377</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Messages</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>751</width>
        <height>251</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QListWidget" name="MinboxListWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>731</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>0</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Inbox</string>
       </property>
      </widget>
     </widget>
     <widget class="QTextEdit" name="Musername">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>Username :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MsendMessageButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>100</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Message</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MbackButton">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>0</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="app-images.qrc">
        <normaloff>:/resources/nack-button</normaloff>:/resources/nack-button</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settings">
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>0</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>761</width>
        <height>431</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>40</y>
         <width>281</width>
         <height>30</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Privacy</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>451</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QComboBox" name="postVIsibility">
          <property name="currentText">
           <string/>
          </property>
          <property name="placeholderText">
           <string>                friends only</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="shownPosts">
          <property name="placeholderText">
           <string>                    friends only</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>who can follow me:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="followPrivacy">
          <property name="placeholderText">
           <string>                            all</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>shown posts:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>post visibility:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="signoutButton">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>390</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Sign Out</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="settingsBackButton">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>0</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="app-images.qrc">
        <normaloff>:/resources/nack-button</normaloff>:/resources/nack-button</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="profile">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>0</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Profile</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="profilePicLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="app-images.qrc">:/resources/profile-icon.webp</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="usernameLabel">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>user_name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>followers</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>70</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>following</string>
      </property>
     </widget>
     <widget class="QLabel" name="numFollowersLabel">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="numFollowingLabel">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>70</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QScrollArea" name="profileFeed">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>99</y>
        <width>751</width>
        <height>371</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>735</width>
         <height>369</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="profile_back_button">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>0</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="app-images.qrc">
        <normaloff>:/resources/nack-button</normaloff>:/resources/nack-button</iconset>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="app-images.qrc"/>
 </resources>
 <connections/>
</ui>
